### ISSUE TITLE
Javascript Error when accessing Script Editor

### ISSUE THREAD

**Issue by Didgeridrew** [home-assistant/frontend#9714]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Whenever I try to edit an existing script using the UI Script Editor I get a Javascript error.  This occurs in every browser I have tried (Chrome, Firefox, Vivaldi) and in the Android companion app.

### Describe the behavior you expected

I expect that the Script Editor would open and allow me to edit scripts through the UI.

### Steps to reproduce the issue

1. Go to Configuration>Scripts
2. Click any "edit script" button

### What version of Home Assistant Core has the issue?

core-2021.7.0-4, core-2021.8.0

### In which browser are you experiencing the issue with?

Vivaldi  4.1.2369.11, Firefox 90.0.2, Chrome Version 92.0.4515.107

### Which operating system are you using to run this browser?

macOS Catalina, 

### Problem-relevant frontend configuration

```yaml
default_config:


frontend:
  themes: !include_dir_merge_named ../themes
```

### Javascript errors shown in your browser console/inspector

```txt
app.30c8e831.js:30594 GET https://j***********ha.duckdns.org/api/config/script/config/test_script_1 500 (Internal Server Error)
I @ app.30c8e831.js:30594
async function (async)
I @ app.30c8e831.js:30594
(anonymous) @ app.30c8e831.js:30594
callApi @ app.30c8e831.js:30594
value @ 7f09d23f.js:2260
E @ app.30c8e831.js:1
performUpdate @ app.30c8e831.js:1
Πq @ app.30c8e831.js:1
async function (async)
Πq @ app.30c8e831.js:1
requestUpdate @ app.30c8e831.js:1
u @ app.30c8e831.js:1
g @ app.30c8e831.js:1
p @ app.30c8e831.js:5232
(anonymous) @ 20a1191a.js:429
r @ 7f09d23f.js:1762
value @ app.30c8e831.js:29505
value @ app.30c8e831.js:29505
value @ app.30c8e831.js:29505
performUpdate @ app.30c8e831.js:1
Πq @ app.30c8e831.js:1
async function (async)
Πq @ app.30c8e831.js:1
requestUpdate @ app.30c8e831.js:1
set @ app.30c8e831.js:1
value @ 3abf3494.js:1
value @ app.30c8e831.js:29505
performUpdate @ app.30c8e831.js:1
Πq @ app.30c8e831.js:1
async function (async)
Πq @ app.30c8e831.js:1
requestUpdate @ app.30c8e831.js:1
set @ app.30c8e831.js:1
value @ app.30c8e831.js:35887
value @ app.30c8e831.js:29505
performUpdate @ app.30c8e831.js:1
Πq @ app.30c8e831.js:1
async function (async)
Πq @ app.30c8e831.js:1
requestUpdate @ app.30c8e831.js:1
set @ app.30c8e831.js:1
W @ app.30c8e831.js:5232
I @ app.30c8e831.js:5232
v @ app.30c8e831.js:5232
_ @ app.30c8e831.js:5232
I @ app.30c8e831.js:5232
O @ app.30c8e831.js:5232
update @ app.30c8e831.js:5232
performUpdate @ app.30c8e831.js:1
Πq @ app.30c8e831.js:1
async function (async)
Πq @ app.30c8e831.js:1
requestUpdate @ app.30c8e831.js:1
set @ app.30c8e831.js:1
W @ app.30c8e831.js:5232
I @ app.30c8e831.js:5232
v @ app.30c8e831.js:5232
_ @ app.30c8e831.js:5232
I @ app.30c8e831.js:5232
O @ app.30c8e831.js:5232
update @ app.30c8e831.js:5232
performUpdate @ app.30c8e831.js:1
Πq @ app.30c8e831.js:1
async function (async)
Πq @ app.30c8e831.js:1
requestUpdate @ app.30c8e831.js:1
set @ app.30c8e831.js:1
t @ app.30c8e831.js:36703
(anonymous) @ app.30c8e831.js:36703
i @ app.30c8e831.js:5232
o @ app.30c8e831.js:5232
(anonymous) @ app.30c8e831.js:36703
```

**Comment by spacegaier** [comment]
I cannot reproduce that on my end on the latest production version. Does it still occur with 2021.9.X for you?

Can you please add your script YAML to the issue. Perhaps it is related to its specific content.

**Comment by Didgeridrew** [comment]
This issue affected every script , but I am closing this issue since it spontaneously resolved itself when I updated to 2021.9
